src/pages/address/_components/WalletDetails.tsx

```diff
@@ -0,0 +1,148 @@
+            data-testid="wallet-balance-value"
+          />
+        </div>
+        {/* hides token field for now */}
+        {detail.tokens !== null && (
+          <div className="flex flex-col gap-y-1 lg:w-[265px] md:w-[294px]">
+            <div
+              data-testid="wallet-tokens-title"
+              className="text-white-700 tracking-[0.01em]"
+            >
+              Tokens
+            </div>
+            <div ref={wrapperRef} className="relative">
+              <div
+                role="button"
+                tabIndex={0}
+                onKeyDown={() =>
+                  onTokenDropDownIconClick(
+                    setIsTokenDropDownIconClicked,
+                    isTokenDropDownIconClicked
+                  )
+                }
+                onClick={() =>
+                  onTokenDropDownIconClick(
+                    setIsTokenDropDownIconClicked,
+                    isTokenDropDownIconClicked
+                  )
+                }
+                className="flex flex-row items-center"
+              >
+                <div className="text-white-50 tracking-[0.01em] mr-[10px]">
+                  {detail.tokens.allTokens.length} Tokens
+                </div>
+                <div>
+                  {isTokenDropDownIconClicked ? (
+                    <FiChevronDown
+                      data-testid="wallet-tokens-dropdown-icon"
+                      size={24}
+                      className="text-white-700"
+                    />
+                  ) : (
+                    <FiChevronUp
+                      data-testid="wallet-tokens-up-icon"
+                      size={24}
+                      className="text-white-700"
+                    />
+                  )}
+                </div>
+                <div>
+                  <NumericFormat
+                    className="text-white-700 ml-[14px] tracking-[0.01em]"
+                    value={detail.tokens.value.toString()}
+                    thousandSeparator
+                    decimalScale={2}
+                    suffix=")"
+                    prefix="($"
+                    data-testid="wallet-tokens-value"
+                  />
+                </div>
+              </div>
+              {isTokenDropDownIconClicked && (
+                <TokenSearchDropDown
+                  data-testid="wallet-token-search-dropdown"
+                  addressTokens={detail.tokens.allTokens}
+                />
+              )}
+            </div>
+          </div>
+        )}
+      </div>
+      <div className="flex flex-col gap-y-1 lg:w-[265px] md:w-[294px]">
+        <div
+          data-testid="wallet-transactions-title"
+          className="text-white-700 tracking-[0.01em]"
+        >
+          Transactions
+        </div>
+        <div
+          data-testid="wallet-transactions-value"
+          className="text-white-50 tracking-[0.01em]"
+        >
+          {useUnitSuffix(detail.transactionCount)}
```

Comment:
Transaction count is 0 👀 
<img width="1363" alt="image" src="https://github.com/WavesHQ/metascan/assets/17063281/47e9952d-f960-499e-82c4-fe527dc88b1f">

Reply:
We can check on a separate PR. Not related to this PR anyway.

Reply:
Yea issue on  API side, weirdly, refetching it again works - assuming it is not syncing with stats properly
1st object
2nd object: refreshes the page 
<img width="614" alt="image" src="https://github.com/WavesHQ/metascan/assets/13292662/e9db15e8-b068-405b-b18a-9a2b3226c3b6">

we can investigate it more in another PR